models/marketing/email_events/email_events.yml

```diff
@@ -20,7 +20,8 @@ models:
       - name: event_id
         description: The ID of the event.
         tests:
-          - unique
+          - dbt_expectations.expect_column_values_to_be_unique:
+              row_condition: "not is_contact_deleted"
```

Comment:
would we ever need to coalesce?

Reply:
That's a great point. My thought was that since we did the coalesce in the staging model then it didn't need to be done here. However, we did the coalesce in the where clause. So, yes a coalesce would be wise to add here. Will update!

Reply:
Updated